### DISCUSSION TITLE
Webhook url be less than 255 characters (url)

### DISCUSSION THREAD

**Discussion by smegoff** [outline/outline#7984] [Self Hosting]
Hi All,

I would like to push notifications via a web hook into teams, however it seems the webhook connector in Teams is quite long and breaks the 255 character limit in Outline. 

Is there a configuration somewhere that increases this limit? The webhook URL in this case is 256 characters. :-(

I might see if I can use power automate to push the webhook, as it uses a shorter path. 

Cheers

Sean

**Comment by tommoor** [comment]
Hi Sean, 255 characters is the limit of the database column – it would require a database migration to increase this I'm afraid

**Reply by smegoff** [reply]
> Hi Sean, 255 characters is the limit of the database column – it would require a database migration to increase this I'm afraid

All good, thank you for answering.
I'm using Zapier now, as for some reason the power automate webhooks don't like the pay load, and consider the array null.

"The execution of template action 'Send_each_adaptive_card' failed: the result of the evaluation of 'foreach' expression '@triggerOutputs()?['body']?['attachments']' is of type 'Null'. The result must be a valid array."

What I am trying to achieve is when a document is updated the team are aware, and they should review the changes.

Am I over thinking this and email notifications go out to permissioned members when a document is updated?

**Reply by tommoor** [reply]
I'm not sure what `['body']?['attachments']` would be here, that doesn't look like it would match any payloads Outline sends.

> Am I over thinking this and email notifications go out to permissioned members when a document is updated?

Possibly, anyone that has collaborated on a document will be subscribed and receive notifications automatically.
